src/main/java/io/appium/java_client/touch/WebElementOption.java

```diff
@@ -0,0 +1,66 @@
+package io.appium.java_client.touch;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+import static java.util.Optional.ofNullable;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.internal.HasIdentity;
+
+import java.util.Map;
+
+public class WebElementOption extends PositionOffsetOption<WebElementOption> {
+
+    private HasIdentity elementId;
+
+    /**
+     * This method creates a build instance of the {@link WebElementOption}
+     *
+     * @param element is the element to calculate offset from.
+     * @param x is the x-offset from the upper left corner of the given element.
+     * @param y is the y-offset from the upper left corner of the given element.
+     * @return the built option
+     */
+    public static WebElementOption elementOption(WebElement element, int x, int y) {
```

Comment:
I'd rather say it is elementOffset. _Option_ suffix it's not needed - let's keep these names short